state/runtime/evm/instructions_test.go

```diff
@@ -31,13 +79,15 @@ func testArithmeticOperation(t *testing.T, f instruction, tests twoOperandsArith
 	s, closeFn := getState()
 	defer closeFn()
 
+	s.config = &allEnabledForks
+
 	for _, i := range tests {
 		s.push(i.b)
 		s.push(i.a)
 
 		f(s)
 
-		assert.EqualValues(t, i.expectedResult, s.pop())
+		assert.EqualValues(t, i.expectedResult.Uint64(), s.pop().Uint64())
```

Comment:
Would assert.Equal do the trick? If you can try that out:
```suggestion
		assert.Equal(t, i.expectedResult, s.pop())
```